internal/run/run.go

```diff
@@ -0,0 +1,23 @@
+package run
+
+import (
+	"log"
+	"time"
+
+	"github.com/NodeFactoryIo/vedran-daemon/internal/lb"
+	"github.com/NodeFactoryIo/vedran-daemon/internal/telemetry"
+	"github.com/go-co-op/gocron"
+)
+
+// Start registers to load balancer and starts sending telemetry
+func Start(client *lb.Client, telemetry telemetry.TelemetryInterface, id string, nodeRPC string, nodeMetrics string, payoutAddress string) error {
+	err := client.Register(id, nodeRPC, payoutAddress, "test-config-hash")
```

Comment:
you should probably change this so it receives polkadot node client with methods to get hash and metrics.
You can pass same client to startSendingTelemetry method

Reply:
Config hash and metrics client are different as they are not on the same url
But I can currently pass metrics client as I have it and will add rpc client when we we have
config hash implementation

Reply:
But that doesn't really matter? You can hide that those are two different urls inside PolkadotNodeService?